chromium_src/android_webview/browser/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
```

Comment:
please make sure to fix all these issues:

https://gist.github.com/goodov/8b3ead13ff49b37c76999f37288128a6

(keeping it as a comment on first DEPS just to make it resolvable and block merge until resolved).